frontend/packages/integration-tests-cypress/support/selectors.ts

```diff
@@ -7,7 +7,7 @@ declare global {
       byLegacyTestID(selector: string): Chainable<Element>;
       byButtonText(selector: string): Chainable<Element>;
       byDataID(selector: string): Chainable<Element>;
-      byTestSelector(selector: string): Chainable<Element>;
+      byTestSelector(selector: string, options?: object): Chainable<Element>;
```

Comment:
We should be using the same type def that Cypress uses for the `options` arg in `cy.get`

Reply:
updated, thanks